tests/test_weave/test_autopatcher.py

```diff
@@ -42,7 +42,8 @@ def hello_world():
     eval(hello_world, model="mockllm/model")
 
     calls = list(patch_weave_client_in_hooks.get_test_calls())
-    assert len([call.name for call in calls]) == 8
+
+    assert len([call.name for call in calls]) == 9
 
     # check for inspect AI patched calls
     assert "sample" in calls[1].name
```

Comment:
could you also add something here to test the the scorer call in particular is present?